_docs/guides/bookinfo.md

```diff
@@ -189,16 +189,15 @@ To start the application, follow the instructions below corresponding to your Is
 1. Bring up the application containers.
 
     1. To test with Consul, run the following command:
-        ```bash
-        docker-compose -f samples/bookinfo/consul/bookinfo.yaml up -d
-        docker-compose -f samples/bookinfo/consul/bookinfo.sidecars.yaml up -d
-        ```
+       ```bash
+       docker-compose -f samples/bookinfo/consul/bookinfo.yaml up -d
+       docker-compose -f samples/bookinfo/consul/bookinfo.sidecar.yaml up -d
```

Comment:
aren’t you reverting a previous fix here ? (plural)

Reply:
Good catch, that wasn't quite right.